src/components/cards/utils/constants.ts

```diff
@@ -0,0 +1,152 @@
+        ? [globalColors.yellow70, '#FFCB0F']
+        : ['#FFCB0F', '#FFDA24'],
+      shadowColor: '#FFCB0F',
+      orbColorLight: globalColors.yellow20,
+      orbColorDark: isDarkMode ? globalColors.yellow80 : globalColors.yellow70,
+      primaryTextColor: globalColors.orange100,
+      secondaryTextColor: globalColors.yellow90,
+    },
+    green: {
+      gradient: isDarkMode
+        ? [globalColors.green70, globalColors.green60]
+        : [globalColors.green60, '#3ECF5B'],
+      shadowColor: globalColors.green60,
+      orbColorLight: globalColors.green20,
+      orbColorDark: isDarkMode ? globalColors.green80 : globalColors.green70,
+      primaryTextColor: globalColors.white100,
+      secondaryTextColor: globalColors.green20,
+    },
+    darkGreen: {
+      gradient: isDarkMode
+        ? [globalColors.green100, globalColors.green90]
+        : [globalColors.green90, globalColors.green80],
+      shadowColor: globalColors.green90,
+      orbColorLight: globalColors.green20,
+      orbColorDark: isDarkMode ? globalColors.green80 : globalColors.green70,
+      primaryTextColor: globalColors.white100,
+      secondaryTextColor: globalColors.green20,
+    },
+  };
+};
+
+export const getLearnCardColorway: (
+  category: LearnCategory,
+  isDarkMode: boolean
+) => CardColorway = (category: LearnCategory, isDarkMode: boolean) => {
+  const colorways = getCardColorways(isDarkMode);
+  switch (category) {
+    case LearnCategory.Essentials:
+      return colorways.purple;
+    case LearnCategory.Safety:
+      return colorways.pink;
+    case LearnCategory.Rainbow:
+      return colorways.blue;
+    case LearnCategory.BeginnersGuides:
+      return colorways.yellow;
+    case LearnCategory.Blockchains:
+      return colorways.darkGreen;
+    case LearnCategory.Web3:
+      return colorways.green;
+    default:
+      return colorways.darkBlue;
+  }
+};
+
+const getStartedCard: LearnCardDetails = {
+  url: 'https://learn.rainbow.me/get-started-with-rainbow',
+  category: LearnCategory.Essentials,
+  key: 'get_started',
+  emoji: '🌈',
+};
+
+const backupsCard: LearnCardDetails = {
+  url: 'https://learn.rainbow.me/the-importance-of-backups',
+  category: LearnCategory.Essentials,
+  key: 'backups',
+  emoji: '☮️',
+};
+
+const protectWalletCard: LearnCardDetails = {
+  url: 'https://learn.rainbow.me/protect-your-wallet',
+  category: LearnCategory.Safety,
+  key: 'protect_wallet',
+  emoji: '🔒',
+};
+
+const connectToDappCard: LearnCardDetails = {
+  url: 'https://learn.rainbow.me/connect-to-a-website-or-app',
+  category: LearnCategory.Essentials,
+  key: 'connect_to_dapp',
+  emoji: '🔌',
+};
+
+const avoidScamsCard: LearnCardDetails = {
+  url: 'https://learn.rainbow.me/avoid-crypto-scams',
+  category: LearnCategory.Safety,
+  key: 'avoid_scams',
+  emoji: '🤬',
+};
+
+export const learnCards: LearnCardDetails[] = [
+  getStartedCard,
+  backupsCard,
+  protectWalletCard,
+  connectToDappCard,
+  avoidScamsCard,
+];
```

Comment:
Do we still need any of this? Seems like it should just live on the screen where we include them. Then `LearnCardDetails` could live with the `LearnCard` too, which might be good

Reply:
do you mean pass in `url`, `category` etc as props to `LearnCard`? or keep the individual cards like `avoidScamsCard` and pass that in as a prop

Reply:
or do you mean define all these `LearnCardDetails` objects in the file that renders `<LearnCard />`? to me that would seem cluttered

Reply:
Yeah, bit of a nit, what I mean is, on the screen where `LearnCard` is used, we could just do:

```jsx
<LearnCard url='https://...' category={LearnCategory.Safety} ... />
```

And if we need `learnCards[]` on a screen, it should just be defined there.

Maintaining copy/config away from it's usage just seems redundant to me, but maybe I'm missing the other usage possibilities atm.

Reply:
As for the types, if I'm looking for "types for the Learn Card", it's nice to be able to go to the `LearnCard` file and have them alongside the component. Main reason not to define related things in the same file is to avoid circular imports, which I don't think is much of an issue in RN (just like the web).

But yeah not a huge deal, not worth blocking this rn 👍

Reply:
ah ok yeah that makes sense. i'll just keep it in this file for now until the learn cards are actually rendered somewhere